### PR TITLE
docs: add contributing file

### PR DESCRIPTION
**Pull request by adamaltman**
We noticed an outdated Google doc. Moving information to the appropriate location and removing unnecessary information.

### CODE REVIEW

**Comment by adamaltman** [comment]
@slavcodev 👋

File: openapi/paths/README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Paths
 =====
 
+* Paths are plural.
```
Comment (adamaltman) [inline openapi/paths/README.md:4]: This should be enforceable by the linter: 

https://redoc.ly/docs/cli/resources/built-in-rules/#path-segment-plural (we should test it and see if we hit any issues)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+- Set up [your environment](./README.md#environment-setup).
+- Use [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow) (open a pull request for any change).
+- Redocly OpenAPI-CLI runs our `lint` and `bundle` command which are controlled through the `.redocly.yaml` configuration file when a pull request is opened (you will see the green checks or red x marks). Merge here just before merging and releasing to production.
+
+Try to set up lint rules to enforce any design rules to keep this list to a minimum (and remove from this list as rules are enabled to automate these checks):
+
+### Schemas
+
+- The schema filename should be singular and start with an uppercase letter (`Customer.yaml`).
```
Comment (adamaltman) [inline CONTRIBUTING.md:19]: I think we could make a lint rule for this (somehow).

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+- Re-use schemas by reference objects (`$ref`). Some commonly used include:
+    - `ResourceId`
+    - `ServerTimestamp`
+- Most APIs have a `_links` property (with at least "self").
+- If a resource has nested objects, consider separating them with the reference objects (`$ref`) if they have potential for reuse.
+- You may organize schemas into sub-folders when appropriate.
+
+### Operations
+
+- Follow [path conventions](./paths/README.md)
```
Comment (swilczynski) [inline CONTRIBUTING.md:33]: Shouldn't this be `./openapi/paths/README.md` ? Current link gives me 404

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+- Use appropriate HTTP methods.
+    - POST to insert new Resource into another.
+    - PUT to create with specified ID or replace existing Resource (must return 200 for updated and 201 for created).
```
Comment (adamaltman) [inline CONTRIBUTING.md:36]: I think we could have a custom lint rule to enforce that PUT requests have both 200 and 201 responses defined.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+- Response must contain http header `Location` of the newly created resource; if the status is 201, the `Location` header must exist.
+
+#### Collections
+
+- Add `"#/parameters/collectionLimit"` and `"#/parameters/collectionOffset"` to Resource collection, if pagination is needed.
+- Add `"#/parameters/collectionFields"` to Resource collection to allow return only specified fields.
+- Add `"#/parameters/collectionExpand"` to Resource collection if Resource contains embedded resources
+- Add `"#/parameters/collectionQuery"` to Resource collection if Resource support search by ‘?q=’ param
+- Add `"#/parameters/collectionFilter"` to Resource collection if Resource support search by ‘?filter=’ param
+- Add `"#/parameters/collectionCriteria"` to Resource collection if Resource support search by ‘?criteria=’ param
```
Comment (colinbird) [inline CONTRIBUTING.md:48]:
```suggestion
- Add `"#/parameters/collectionExpand"` to Resource collection if Resource contains embedded resources.
- Add `"#/parameters/collectionQuery"` to Resource collection if Resource support search by ‘?q=’ param.
- Add `"#/parameters/collectionFilter"` to Resource collection if Resource support search by ‘?filter=’ param.
- Add `"#/parameters/collectionCriteria"` to Resource collection if Resource support search by ‘?criteria=’ param.
```

File: README.md [hunk 6]
```diff
@@ -12,7 +12,11 @@ TBD: Add Redocly validation status badge.
 - [Reports API docs](https://reports-api-docs.rebilly.com/)
 - [Storefront API docs](https://storefront-api-docs.rebilly.com/) (this API is in development and subject to change)
 
-## Working on the API definitions
+
+## Contributing
+
+[Read our contribution guide](./CONTRIBUTING.md).
+## Environment setup
```
Comment (nanobash) [inline README.md:19]: Maybe insert line spacing between content and the title to be consistent with rest of the styling

**Review by ashkarpetin — APPROVED** [review]

**Review by colinbird — APPROVED** [review]

**Review by creocoder — APPROVED** [review]

**Pull request merged**